dll/win32/shell32/folders/CFSFolder.cpp

```diff
@@ -528,11 +528,11 @@ CFSFolder::~CFSFolder()
 }
 
 static const shvheader GenericSFHeader[] = {
-    {IDS_SHV_COLUMN_NAME, SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 15},
+    {IDS_SHV_COLUMN_NAME, SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 18},
```

Comment:
What was the magic 15? What is the magic 18?

Reply:
The default width of the column.